connect/reports/renderers/xlsx.py

```diff
@@ -29,13 +31,14 @@ def render(self, data, output_file):
                 self.template,
             ),
         )
-
         ws = wb['Data']
         for row in data:
             for col_idx, cell_value in enumerate(row, start=start_col_idx):
                 ws.cell(row_idx, col_idx, value=cell_value)
             row_idx += 1
-        self._add_info_sheet(wb.create_sheet('Info'), start_time)
+
+        self._add_info_sheet(wb.create_sheet('Info'), datetime.now(tz=pytz.utc))
```

Comment:
Pass here the start time makes no sense since the report has already been generated.
You can set it as an instance member of the renderer (i.e. in render method -> self.start_time = start_time or datetime.now(...)) and then retrieve it in the _add_info_sheet method.

Reply:
True, I'm gonna change it!